forge.config.js

```diff
@@ -65,7 +65,6 @@ const config = {
             name: '@electron-forge/maker-dmg',
             config: {
                 format: 'ULFO',
-                name: 'chaiNNer',
```

Comment:
is there a reason you removed this?

Reply:
Yes, when the name is set to chaiNNEr the resulting DMG is always called `chaiNNer.dmg`, regardless of the architecture it is built for.

So building arm64 creates a DMG called `chaiNNEr.dmg`. When the x64 version is built, it also creates a DMG called `chaiNNEr.dmg`. Which basically overwrites the arm64 build when running `npm run make-mac-all`.

By not providing a name, the DMGs are named automatically: `chaiNNer-<version>-<architecture>.dmg`. 
Example for the current version:
```
chaiNNer-0.19.0-arm64.dmg
chaiNNer-0.19.0-x64.dmg
```

Reply:
Oh cool